### PR TITLE
feat(Icon): Added infoOutlineIcon

### PR DESCRIPTION
**Pull request by yogesh2490**
Added info-outline svg file

### CODE REVIEW

File: info-outline.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg fill="#9E9E9E" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```
Comment (haronius) [inline info-outline.svg]: We should pass in the fill as a prop so we can change the color in the future

File: src/components/icons/InfoOutlineIcon.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const InfoOutlineIcon = () => ({
+  render() {
+    return (
+      <div>
+        <svg fill="#9E9E9E" height='24px' viewBox="0 0 24 24" width='24px'>
```
Comment (haronius) [inline src/components/icons/InfoOutlineIcon.js]: Also should be a prop

File: info-outline.svg (hunk 1 above)
Comment (haronius) [reply info-outline.svg]: Sizing should be based on props as well. Although you could add a default of 24px

Comment (seanevd) [inline info-outline.svg]: I'm not really sure why you are including this .svg at all. You only need to put it in src/components/icons. You also need to make sure you are exporting it in src/components/icons/index.js so you can use it when you import it from insidesales-components.

File: src/components/icons/InfoOutlineIcon.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const InfoOutlineIcon = () => ({
+  render() {
+    return (
+      <div>
```
Comment (seanevd) [inline src/components/icons/InfoOutlineIcon.js]: There's no reason to wrap this in a div, unless you have a good one.

File: src/components/icons/InfoOutlineIcon.js (hunk 2 above)
Comment (seanevd) [reply src/components/icons/InfoOutlineIcon.js]: `<svg {...props.size || { width: '24px', height: '24px' }} {...props} viewBox="0 0 24 24">` Gives a default height and width. Allows custom size, if needed.

Comment (seanevd) [inline src/components/icons/InfoOutlineIcon.js]: Do not assign a fill color directly on the svg. You can pass the fill color as a prop where you import it.

File: info-outline.svg (hunk 1 above)
Comment (yogesh2490) [reply info-outline.svg]: this should not be here you are right. It was my bad

File: src/components/icons/InfoOutlineIcon.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const InfoOutlineIcon = props => (
+  <svg {...props.fill } {...props.size || { width: '24px', height: '24px' }} {...props} viewBox="0 0 24 24">
```
Comment (seanevd) [inline src/components/icons/InfoOutlineIcon.js]: You don't need `{...props.fill}`, `{...props}` should take care of that.

**Comment by seanevd** [comment]
I think the last thing you need to do is include it as an import in icons/index.js

**Pull request merged**